ext/compressors/lz4/CMakeLists.txt

```diff
@@ -57,7 +57,10 @@ endif()
 
 if(HAVE_BUILTIN_EXTENSION_LZ4 OR ENABLE_LZ4)
     add_library(wiredtiger_lz4 ${link_type} ${sources})
-    target_link_libraries(wiredtiger_lz4 "lz4")
+    # HAVE_LIBLZ4 contains the full pathname to the lib, which is
+    # what the cmake docs recommend using
+    #target_link_libraries(wiredtiger_lz4 "lz4")
```

Comment:
Can remove line

Reply:
Indeed, I'll push another commit that gets rid of these. I also just noticed that I forgot to make this change for the zlib extension (on my machine zlib is in the default linker path, so it didn't fail, but that isn't always true), so I'll push that as well.